src/services/Item/index.ts

```diff
@@ -0,0 +1,25 @@
+import Axios from 'axios';
+import { IncomingMessage } from 'http';
+
+import { listConfig, actionConfig } from './config';
+import { ItemListParams, ItemActionRequestParams } from '@src/types';
+import { IItem } from '@src/interfaces';
+
+const list = async (
+  params: ItemListParams & { userId?: number },
+  req?: IncomingMessage
+): Promise<IItem[]> => Axios(listConfig(params, req)).then((res) => res.data);
+
+const action = async (
+  ids: number[],
+  body: ItemActionRequestParams,
+  req?: IncomingMessage
+): Promise<IItem> =>
+  Axios(actionConfig(ids, body, req)).then((res) => res.data);
```

Comment:
```suggestion
  axios(actionConfig(ids, body, req)).then((res) => res.data);
```

다른 모든 코드에서 axios의 첫 글자를 소문자로 사용하고 있습니다.

Reply:
e418508